CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 All notable changes to this project will be documented in this file. This project adheres to Semantic Versioning.
 
-### 7.0.0-RC1
+### 7.0.0
```

Comment:
```suggestion
### 7.0.0
* Removes queue processor indexer. Now there is a single indexer, which simply sends product ID's to the message queue with an action to either update or delete products that will be consumed by SyncService
* By removing the indexer, fixes an issue with duplicated rows, which caused the indexer to run multiple times for the same product
* By removing the indexer, also fixes an issue with out of memory when trying to index large catalogs
* By removing the indexer, also fixes an issue with MassProductAttributeUpdate operations, which would cause the index queue to be over populated when indexer was on save mode
```